### PR TITLE
Remove some TODOs

### PR DESCRIPTION
**Pull request by Mouvedia**
> Which issue, if any, is this issue related to?

see tasklist on #7060

> Is there anything in the PR that needs further explanation?

I didn't tick the review because I couldn't resolve all the TODOs; 6 files are left:

- `lib/rules/functionCommaSpaceChecker.js`
- `lib/rules/color-named/index.js`
- <del>`lib/rules/selector-class-pattern/index.js`</del>
- `lib/rules/selector-descendant-combinator-no-non-space/index.js`
- `lib/rules/string-quotes/index.js`
- v16:
  - `lib/formatters/__tests__/terminalLink.test.mjs`
  - `.github/dependabot.yml`

### CODE REVIEW

File: lib/rules/string-quotes/index.js [hunk 1]
```diff
@@ -191,7 +191,6 @@ const rule = (primary, secondaryOptions, context) => {
 
 			if (isAtRule(node) && node.name === 'charset') {
 				// allow @charset rules to have double quotes, in spite of the configuration
-				// TODO: @charset should always use double-quotes, see https://github.com/stylelint/stylelint/issues/2788
```
Comment (ybiquitous) [inline lib/rules/string-quotes/index.js:194]: Could you revert this change since `string-quotes` has been removed in the `v16` branch?

**Review by ybiquitous — CHANGES_REQUESTED** [review]
@Mouvedia Thanks for the pull request. I've commented on `string-quotes`, but LGTM for others 👍🏼

File: lib/rules/string-quotes/index.js (hunk 1 above)
Comment (Mouvedia) [reply lib/rules/string-quotes/index.js:194]: Iv created an issue on elirasza/stylelint-stylistic.
Since this won't be ever be tackled by us, the TODO can be safely removed.

Comment (ybiquitous) [reply lib/rules/string-quotes/index.js:194]: If we changed this file, we would need to resolve conflicts when rebasing or merging into the `v16` branch. It's just a pain, and I'd like to avoid extra work (see #7002).

Comment (Mouvedia) [reply lib/rules/string-quotes/index.js:194]: I see; this sucks.
Hopefully the ESM migration that you—for the most part—undertook, will come to an end soon.
Ill fix that later on today.

**Comment by Mouvedia** [comment]
@ybiquitous Ill leave the v16 related ones for you to clean up :)

**Review by ybiquitous — APPROVED** [review]
Thanks! 👍🏼
